### PR TITLE
mds: initialize cap_revoke_eviction_timeout with conf

### PR DESCRIPTION
**Pull request by simon-rock**
Signed-off-by: simon gao <simon29rock@gmail.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by simon-rock** [comment]
@tchaikov  review this please,thx

File: src/common/legacy_config_opts.h [hunk 1]
```diff
@@ -413,6 +413,7 @@ OPTION(mds_freeze_tree_timeout, OPT_FLOAT)    // detecting freeze tree deadlock
 OPTION(mds_health_summarize_threshold, OPT_INT) // collapse N-client health metrics to a single 'many'
 OPTION(mds_reconnect_timeout, OPT_FLOAT)  // seconds to wait for clients during mds restart
 	      //  make it (mdsmap.session_timeout - mds_beacon_grace)
+OPTION(mds_cap_revoke_eviction_timeout, OPT_DOUBLE)  // evict client session if it cann't respone revoke msg
```
Comment (batrick) [inline src/common/legacy_config_opts.h]: We do not change this file anymore. Please remove this change.

File: src/mds/Server.cc [hunk 2]
```diff
@@ -193,6 +193,7 @@ Server::Server(MDSRank *m) :
   terminating_sessions(false),
   recall_throttle(g_conf().get_val<double>("mds_recall_max_decay_rate"))
 {
+  cap_revoke_eviction_timeout = g_conf()->mds_cap_revoke_eviction_timeout;
```
Comment (batrick) [inline src/mds/Server.cc]: Should be:

`cap_revoke_eviction_timeout = g_conf().get_val<double>("mds_cap_revoke_eviction_timeout");`

**Comment by batrick** [comment]
I suggest using the PR title when amending your commit message.

**Comment by batrick** [comment]
Thanks for finding this by the way!

**Comment by simon-rock** [comment]
thx. Our branch is based on Luminous.;)

File: src/common/legacy_config_opts.h (hunk 1 above)
Comment (simon-rock) [reply src/common/legacy_config_opts.h]: ok

File: src/mds/Server.cc (hunk 2 above)
Comment (simon-rock) [reply src/mds/Server.cc]: ok. thx

**Pull request merged**